src/chart/model/scaling/viewport.model.ts

```diff
@@ -152,6 +152,15 @@ export abstract class ViewportModel extends ChartBaseElement implements Viewable
 		share(),
 	);
 	//endregion
+
+	protected doActivate(): void {
+		super.doActivate();
```

Comment:
here I deleted: `this.changed = new Subject()`
because of this line Yaxis was broken